src/NHSD.BuyingCatalogue.API/Program.cs

```diff
@@ -21,9 +20,9 @@ public static int Main(string[] args)
                 .Enrich.FromLogContext()
 #if DEBUG
                 .WriteTo.Debug()
-                .WriteTo.Console(new RenderedCompactJsonFormatter())
+                .WriteTo.Console()
```

Comment:
I don't we need the conditional compilation, now that we have changed the render option.